diff --git a/node_modules/expo-modules-core/android/CMakeLists.txt b/node_modules/expo-modules-core/android/CMakeLists.txt
index 5d309ef..54af85b 100644
--- a/node_modules/expo-modules-core/android/CMakeLists.txt
+++ b/node_modules/expo-modules-core/android/CMakeLists.txt
@@ -126,6 +126,7 @@ target_link_libraries(
   fbjni::fbjni
   ReactAndroid::jsi
   android
+  c++_shared
   ${JSEXECUTOR_LIB}
   ${NEW_ARCHITECTURE_DEPENDENCIES}
 )
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/.gradle/8.14.3/executionHistory/executionHistory.bin b/node_modules/expo-modules-core/expo-module-gradle-plugin/.gradle/8.14.3/executionHistory/executionHistory.bin
new file mode 100644
index 0000000..d9dc284
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/.gradle/8.14.3/executionHistory/executionHistory.bin differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/.gradle/8.14.3/executionHistory/executionHistory.lock b/node_modules/expo-modules-core/expo-module-gradle-plugin/.gradle/8.14.3/executionHistory/executionHistory.lock
new file mode 100644
index 0000000..ce2d294
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/.gradle/8.14.3/executionHistory/executionHistory.lock differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/.gradle/buildOutputCleanup/buildOutputCleanup.lock b/node_modules/expo-modules-core/expo-module-gradle-plugin/.gradle/buildOutputCleanup/buildOutputCleanup.lock
new file mode 100644
index 0000000..f82461e
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/.gradle/buildOutputCleanup/buildOutputCleanup.lock differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/.gradle/buildOutputCleanup/cache.properties b/node_modules/expo-modules-core/expo-module-gradle-plugin/.gradle/buildOutputCleanup/cache.properties
new file mode 100644
index 0000000..24edae2
--- /dev/null
+++ b/node_modules/expo-modules-core/expo-module-gradle-plugin/.gradle/buildOutputCleanup/cache.properties
@@ -0,0 +1,2 @@
+#Wed Mar 11 10:58:29 IST 2026
+gradle.version=8.14.3
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/.gradle/buildOutputCleanup/outputFiles.bin b/node_modules/expo-modules-core/expo-module-gradle-plugin/.gradle/buildOutputCleanup/outputFiles.bin
new file mode 100644
index 0000000..9955ff4
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/.gradle/buildOutputCleanup/outputFiles.bin differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/.gradle/file-system.probe b/node_modules/expo-modules-core/expo-module-gradle-plugin/.gradle/file-system.probe
new file mode 100644
index 0000000..f8b0fc5
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/.gradle/file-system.probe differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/bin/main/expo/modules/plugin/AutolinkingIntegration.kt b/node_modules/expo-modules-core/expo-module-gradle-plugin/bin/main/expo/modules/plugin/AutolinkingIntegration.kt
new file mode 100644
index 0000000..40f3101
--- /dev/null
+++ b/node_modules/expo-modules-core/expo-module-gradle-plugin/bin/main/expo/modules/plugin/AutolinkingIntegration.kt
@@ -0,0 +1,10 @@
+package expo.modules.plugin
+
+import org.gradle.api.Project
+import java.io.File
+
+interface AutolinkingIntegration {
+  fun getExpoDependency(project: Project, name: String): Any
+
+  fun getShouldUsePublicationScriptPath(project: Project): File?
+}
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/bin/main/expo/modules/plugin/AutolinkingIntegrationImpl.kt b/node_modules/expo-modules-core/expo-module-gradle-plugin/bin/main/expo/modules/plugin/AutolinkingIntegrationImpl.kt
new file mode 100644
index 0000000..3d3ea04
--- /dev/null
+++ b/node_modules/expo-modules-core/expo-module-gradle-plugin/bin/main/expo/modules/plugin/AutolinkingIntegrationImpl.kt
@@ -0,0 +1,53 @@
+package expo.modules.plugin
+
+import expo.modules.plugin.configuration.ExpoAutolinkingConfig
+import expo.modules.plugin.configuration.GradleProject
+import org.gradle.api.Project
+import java.io.File
+
+class AutolinkingIntegrationImpl : AutolinkingIntegration {
+  override fun getExpoDependency(project: Project, name: String): Any {
+    val config = getProjectConfig(project, name)
+
+    if (config.usePublication) {
+      val publication = requireNotNull(config.publication)
+      return "${publication.groupId}:${publication.artifactId}:${publication.version}"
+    }
+
+    return project.rootProject.findProject(":$name")
+      ?: throw IllegalStateException("Couldn't find project with name $name.")
+  }
+
+  override fun getShouldUsePublicationScriptPath(project: Project): File? {
+    val config = getProjectConfig(project)
+    val scriptPath = config.shouldUsePublicationScriptPath
+      ?: return null
+    val scriptFile = File(scriptPath)
+
+    if (!scriptFile.exists()) {
+      project.logger.warn("[ExpoAutolinkingPlugin] The script file does not exist: $scriptFile")
+      return null
+    }
+
+    return scriptFile
+  }
+
+  private fun getProjectConfig(project: Project, projectName: String): GradleProject {
+    val config = getConfig(project)
+    val dependency = config.allProjects.find { it.name == projectName }
+
+    if (dependency == null) {
+      throw IllegalStateException("Couldn't find project with name $projectName in `expo-autolinking-settings` configuration.")
+    }
+
+    return dependency
+  }
+
+  private fun getProjectConfig(project: Project) = getProjectConfig(project, project.name)
+
+  private fun getConfig(project: Project): ExpoAutolinkingConfig {
+    val gradleExtension = project.gradle.extensions.findByType(ExpoGradleExtension::class.java)
+      ?: throw IllegalStateException("`ExpoGradleExtension` not found. Please, make sure that `useExpoModules` was called in `settings.gradle`.")
+    return gradleExtension.config
+  }
+}
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/bin/main/expo/modules/plugin/ExpoModulesGradlePlugin.kt b/node_modules/expo-modules-core/expo-module-gradle-plugin/bin/main/expo/modules/plugin/ExpoModulesGradlePlugin.kt
new file mode 100644
index 0000000..71d54fc
--- /dev/null
+++ b/node_modules/expo-modules-core/expo-module-gradle-plugin/bin/main/expo/modules/plugin/ExpoModulesGradlePlugin.kt
@@ -0,0 +1,53 @@
+// Copyright 2015-present 650 Industries. All rights reserved.
+
+package expo.modules.plugin
+
+import com.android.build.api.variant.AndroidComponentsExtension
+import expo.modules.plugin.gradle.ExpoGradleHelperExtension
+import expo.modules.plugin.gradle.ExpoModuleExtension
+import org.gradle.api.Plugin
+import org.gradle.api.Project
+import org.gradle.internal.extensions.core.extra
+
+private val lock = Any()
+
+abstract class ExpoModulesGradlePlugin : Plugin<Project> {
+  override fun apply(project: Project) {
+    val kotlinVersion = getKotlinVersion(project)
+    val kspVersion = getKSPVersion(project)
+
+    // Creates a user-facing extension that provides access to the `ExpoGradleHelperExtension`.
+    val expoModuleExtension = project.extensions.create("expoModule", ExpoModuleExtension::class.java, project)
+
+    with(project) {
+      applyDefaultPlugins()
+      applyKotlin(kotlinVersion, kspVersion)
+      applyDefaultDependencies()
+      applyDefaultAndroidSdkVersions()
+
+      extensions.getByType(AndroidComponentsExtension::class.java).finalizeDsl {
+        applyPublishing(expoModuleExtension)
+      }
+    }
+
+    // Adds the expoGradleHelper extension to the gradle instance if it doesn't exist.
+    // If it does exist, that means it was added by a different project.
+    synchronized(lock) {
+      with(project.gradle.extensions) {
+        if (findByType(ExpoGradleHelperExtension::class.java) == null) {
+          create("expoGradleHelper", ExpoGradleHelperExtension::class.java)
+        }
+      }
+    }
+  }
+
+  private fun getKotlinVersion(project: Project): String {
+    return project.rootProject.extra.safeGet<String>("kotlinVersion")
+      ?: project.logger.warnIfNotDefined("kotlinVersion", "2.0.21")
+  }
+
+  private fun getKSPVersion(project: Project): String {
+    return project.rootProject.extra.safeGet<String>("kspVersion")
+      ?: project.logger.warnIfNotDefined("kspVersion", "2.0.21-1.0.28")
+  }
+}
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/bin/main/expo/modules/plugin/ExtraPropertiesExtension.kt b/node_modules/expo-modules-core/expo-module-gradle-plugin/bin/main/expo/modules/plugin/ExtraPropertiesExtension.kt
new file mode 100644
index 0000000..b0d212e
--- /dev/null
+++ b/node_modules/expo-modules-core/expo-module-gradle-plugin/bin/main/expo/modules/plugin/ExtraPropertiesExtension.kt
@@ -0,0 +1,13 @@
+// Copyright 2015-present 650 Industries. All rights reserved.
+
+package expo.modules.plugin
+
+import org.gradle.api.plugins.ExtraPropertiesExtension
+
+internal inline fun <reified T> ExtraPropertiesExtension.safeGet(name: String): T? {
+  return if (has(name)) {
+    get(name) as? T
+  } else {
+    null
+  }
+}
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/bin/main/expo/modules/plugin/ProjectConfiguration.kt b/node_modules/expo-modules-core/expo-module-gradle-plugin/bin/main/expo/modules/plugin/ProjectConfiguration.kt
new file mode 100644
index 0000000..7c22069
--- /dev/null
+++ b/node_modules/expo-modules-core/expo-module-gradle-plugin/bin/main/expo/modules/plugin/ProjectConfiguration.kt
@@ -0,0 +1,102 @@
+// Copyright 2015-present 650 Industries. All rights reserved.
+
+package expo.modules.plugin
+
+import com.android.build.gradle.LibraryExtension
+import expo.modules.plugin.android.PublicationInfo
+import expo.modules.plugin.android.applyLinterOptions
+import expo.modules.plugin.android.applyPublishingVariant
+import expo.modules.plugin.android.applySDKVersions
+import expo.modules.plugin.android.createEmptyExpoPublishTask
+import expo.modules.plugin.android.createEmptyExpoPublishToMavenLocalTask
+import expo.modules.plugin.android.createExpoPublishTask
+import expo.modules.plugin.android.createExpoPublishToMavenLocalTask
+import expo.modules.plugin.android.createReleasePublication
+import expo.modules.plugin.gradle.ExpoModuleExtension
+import org.gradle.api.Project
+import org.gradle.api.publish.PublishingExtension
+import org.gradle.internal.extensions.core.extra
+import java.io.File
+
+internal fun Project.applyDefaultPlugins() {
+  if (!plugins.hasPlugin("com.android.library")) {
+    plugins.apply("com.android.library")
+  }
+  if (!plugins.hasPlugin("kotlin-android")) {
+    plugins.apply("kotlin-android")
+  }
+  if (!plugins.hasPlugin("maven-publish")) {
+    plugins.apply("maven-publish")
+  }
+}
+
+internal fun Project.applyKotlin(kotlinVersion: String, kspVersion: String) {
+  extra.set("kotlinVersion", kotlinVersion)
+  extra.set("kspVersion", kspVersion)
+
+  project.dependencies.add("implementation", "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlinVersion")
+}
+
+internal fun Project.applyDefaultDependencies() {
+  val modulesCore = rootProject.project(":expo-modules-core")
+  if (project != modulesCore) {
+    project.dependencies.add("compileOnly", modulesCore)
+
+    project.dependencies.add("testImplementation", modulesCore)
+    project.dependencies.add("androidTestImplementation", modulesCore)
+  }
+}
+
+internal fun Project.applyDefaultAndroidSdkVersions() {
+  with(androidLibraryExtension()) {
+    applySDKVersions(
+      compileSdk = rootProject.extra.safeGet("compileSdkVersion")
+        ?: logger.warnIfNotDefined("compileSdkVersion", 36),
+      minSdk = rootProject.extra.safeGet("minSdkVersion")
+        ?: logger.warnIfNotDefined("minSdkVersion", 24),
+      targetSdk = rootProject.extra.safeGet("targetSdkVersion")
+        ?: logger.warnIfNotDefined("targetSdkVersion", 36)
+    )
+    applyLinterOptions()
+  }
+}
+
+/**
+ * Applies the necessary configuration for publishing to the local Maven repository.
+ * It need to be called when DSL is finalized.
+ */
+internal fun Project.applyPublishing(expoModulesExtension: ExpoModuleExtension) {
+  if (!expoModulesExtension.canBePublished) {
+    createEmptyExpoPublishTask()
+    createEmptyExpoPublishToMavenLocalTask()
+    return
+  }
+
+  val libraryExtension = androidLibraryExtension()
+
+  libraryExtension
+    .applyPublishingVariant()
+
+  afterEvaluate {
+    val publicationInfo = PublicationInfo(this)
+
+    publishingExtension()
+      .publications
+      .createReleasePublication(publicationInfo)
+
+    createExpoPublishToMavenLocalTask(publicationInfo, expoModulesExtension)
+
+    val npmLocalRepositoryRelativePath = "local-maven-repo"
+    val npmLocalRepository = File("${project.projectDir.parentFile}/${npmLocalRepositoryRelativePath}").toURI()
+    publishingExtension().repositories.mavenLocal { mavenRepo ->
+      mavenRepo.name = "NPMPackage"
+      mavenRepo.url = npmLocalRepository
+    }
+
+    createExpoPublishTask(publicationInfo, expoModulesExtension, npmLocalRepositoryRelativePath)
+  }
+}
+
+internal fun Project.androidLibraryExtension() = extensions.getByType(LibraryExtension::class.java)
+
+internal fun Project.publishingExtension() = extensions.getByType(PublishingExtension::class.java)
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/bin/main/expo/modules/plugin/Version.kt b/node_modules/expo-modules-core/expo-module-gradle-plugin/bin/main/expo/modules/plugin/Version.kt
new file mode 100644
index 0000000..87f782c
--- /dev/null
+++ b/node_modules/expo-modules-core/expo-module-gradle-plugin/bin/main/expo/modules/plugin/Version.kt
@@ -0,0 +1,30 @@
+package expo.modules.plugin
+
+data class Version(
+  val major: Int,
+  val minor: Int,
+  val patch: Int
+) {
+  fun isAtLeast(other: Version): Boolean {
+    return major > other.major ||
+      (major == other.major && minor > other.minor) ||
+      (major == other.major && minor == other.minor && patch >= other.patch)
+  }
+
+  fun isAtLeast(major: Int, minor: Int, patch: Int): Boolean {
+    return isAtLeast(Version(major, minor, patch))
+  }
+
+  fun toNumber(): Int {
+    return major * 10000 + minor * 100 + patch
+  }
+
+  companion object {
+    fun fromString(value: String): Version {
+      // strip pre-release
+      val normalizedValue = value.substringBefore('-')
+      val (major, minor, patch) = normalizedValue.split(".").map { it.toInt() }
+      return Version(major, minor, patch)
+    }
+  }
+}
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/bin/main/expo/modules/plugin/Warnings.kt b/node_modules/expo-modules-core/expo-module-gradle-plugin/bin/main/expo/modules/plugin/Warnings.kt
new file mode 100644
index 0000000..d6aa129
--- /dev/null
+++ b/node_modules/expo-modules-core/expo-module-gradle-plugin/bin/main/expo/modules/plugin/Warnings.kt
@@ -0,0 +1,13 @@
+package expo.modules.plugin
+
+import org.slf4j.Logger
+
+private val notDefinedKeys = mutableSetOf<String>()
+
+fun <T> Logger.warnIfNotDefined(name: String, defaultValue: T): T = synchronized(notDefinedKeys) {
+  if (!notDefinedKeys.contains(name)) {
+    notDefinedKeys.add(name)
+    warn("Property '$name' is not defined. Using default value: '$defaultValue'. Please ensure the 'expo-root-project' plugin is applied to your root project.")
+  }
+  return defaultValue
+}
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/bin/main/expo/modules/plugin/android/AndroidLibraryExtension.kt b/node_modules/expo-modules-core/expo-module-gradle-plugin/bin/main/expo/modules/plugin/android/AndroidLibraryExtension.kt
new file mode 100644
index 0000000..9bfae8d
--- /dev/null
+++ b/node_modules/expo-modules-core/expo-module-gradle-plugin/bin/main/expo/modules/plugin/android/AndroidLibraryExtension.kt
@@ -0,0 +1,23 @@
+package expo.modules.plugin.android
+
+import com.android.build.gradle.LibraryExtension
+
+internal fun LibraryExtension.applySDKVersions(compileSdk: Int, minSdk: Int, targetSdk: Int) {
+  this.compileSdk = compileSdk
+  defaultConfig {
+    this@defaultConfig.minSdk = minSdk
+    this@defaultConfig.targetSdk = targetSdk
+  }
+}
+
+internal fun LibraryExtension.applyLinterOptions() {
+  lintOptions.isAbortOnError = false
+}
+
+internal fun LibraryExtension.applyPublishingVariant() {
+  publishing { publishing ->
+    publishing.singleVariant("release") {
+      withSourcesJar()
+    }
+  }
+}
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/bin/main/expo/modules/plugin/android/MavenPublicationExtension.kt b/node_modules/expo-modules-core/expo-module-gradle-plugin/bin/main/expo/modules/plugin/android/MavenPublicationExtension.kt
new file mode 100644
index 0000000..951fd97
--- /dev/null
+++ b/node_modules/expo-modules-core/expo-module-gradle-plugin/bin/main/expo/modules/plugin/android/MavenPublicationExtension.kt
@@ -0,0 +1,252 @@
+@file:OptIn(ExperimentalSerializationApi::class)
+
+package expo.modules.plugin.android
+
+import expo.modules.plugin.androidLibraryExtension
+import expo.modules.plugin.gradle.ExpoModuleExtension
+import expo.modules.plugin.publishingExtension
+import groovy.lang.Binding
+import groovy.lang.GroovyShell
+import kotlinx.serialization.ExperimentalSerializationApi
+import kotlinx.serialization.json.Json
+import kotlinx.serialization.json.JsonArray
+import kotlinx.serialization.json.JsonElement
+import kotlinx.serialization.json.JsonObject
+import kotlinx.serialization.json.JsonPrimitive
+import kotlinx.serialization.json.jsonArray
+import kotlinx.serialization.json.jsonObject
+import kotlinx.serialization.json.jsonPrimitive
+import org.gradle.api.Project
+import org.gradle.api.Task
+import org.gradle.api.component.SoftwareComponent
+import org.gradle.api.publish.PublicationContainer
+import org.gradle.api.publish.maven.MavenPublication
+import org.gradle.api.tasks.TaskProvider
+import java.nio.file.Path
+import kotlin.io.path.exists
+import kotlin.io.path.toPath
+
+internal data class PublicationInfo(
+  val components: SoftwareComponent,
+  val groupId: String,
+  val artifactId: String,
+  val version: String,
+) {
+  constructor(
+    project: Project,
+  ) : this(
+    components = project.components.getByName("release"),
+    groupId = project.group.toString(),
+    artifactId = requireNotNull(project.androidLibraryExtension().namespace) {
+      "'android.namespace' is not defined"
+    },
+    version = requireNotNull(project.androidLibraryExtension().defaultConfig.versionName) {
+      "'android.defaultConfig.versionName' is not defined"
+    },
+  )
+
+  fun resolvePath(repositoryPath: Path): Path {
+    val groupPath = groupId.replace('.', '/')
+    val artifactPath = "$groupPath/$artifactId/$version"
+    val publicationPath = repositoryPath.resolve(artifactPath)
+
+    return publicationPath
+  }
+
+  override fun toString(): String {
+    return "$groupId:$artifactId:$version"
+  }
+}
+
+internal fun PublicationContainer.createReleasePublication(publicationInfo: PublicationInfo) {
+  create("release", MavenPublication::class.java) { mavenPublication ->
+    with(mavenPublication) {
+      from(publicationInfo.components)
+      groupId = publicationInfo.groupId
+      artifactId = publicationInfo.artifactId
+      version = publicationInfo.version
+
+      mavenPublication.pom { pom ->
+        pom.name.set(publicationInfo.artifactId)
+        pom.url.set("https://github.com/expo/expo")
+
+        pom.licenses { licenses ->
+          licenses.license { license ->
+            license.name.set("MIT License")
+            license.url.set("https://github.com/expo/expo/blob/main/LICENSE")
+          }
+        }
+
+        pom.scm { scm ->
+          scm.connection.set("https://github.com/expo/expo.git")
+          scm.developerConnection.set("https://github.com/expo/expo.git")
+          scm.url.set("https://github.com/expo/expo")
+        }
+      }
+    }
+  }
+}
+
+internal fun Project.createExpoPublishTask(publicationInfo: PublicationInfo, expoModulesExtension: ExpoModuleExtension, pathToRepository: String): TaskProvider<Task> {
+  val taskProvider = tasks.register("expoPublish") { task ->
+    task.doLast {
+      expoPublishBody(publicationInfo, expoModulesExtension, pathToRepository = pathToRepository)
+    }
+  }
+  taskProvider.configure { task ->
+    val publishTask = tasks.getByName("publish")
+
+    task.group = "publishing"
+    task.description = "Publishes the library to the GitHub Packages repository"
+    task.dependsOn(publishTask)
+  }
+
+  return taskProvider
+}
+
+internal fun Project.createEmptyExpoPublishTask(): TaskProvider<Task> {
+  val taskProvider = tasks.register("expoPublish") { task ->
+    task.doLast {
+      logger.warn("Publishing is not configured for this project!")
+    }
+  }
+  taskProvider.configure { task ->
+    task.group = "publishing"
+    task.description = "Publishes the library to the GitHub Packages repository"
+  }
+
+  return taskProvider
+}
+
+internal fun Project.createEmptyExpoPublishToMavenLocalTask(): TaskProvider<Task> {
+  val taskProvider = tasks.register("expoPublishToMavenLocal") { task ->
+    task.doLast {
+      logger.warn("Publishing is not configured for this project!")
+    }
+  }
+  taskProvider.configure { task ->
+    task.group = "publishing"
+    task.description = "Publishes the library to the local Maven repository"
+  }
+
+  return taskProvider
+}
+
+internal fun Project.createExpoPublishToMavenLocalTask(publicationInfo: PublicationInfo, expoModulesExtension: ExpoModuleExtension): TaskProvider<Task> {
+  val taskProvider = tasks.register("expoPublishToMavenLocal") { task ->
+    task.doLast {
+      expoPublishBody(publicationInfo, expoModulesExtension)
+    }
+  }
+  taskProvider.configure { task ->
+    val publishToMavenLocalTask = tasks.getByName("publishToMavenLocal")
+
+    task.group = "publishing"
+    task.description = "Publishes the library to the local Maven repository"
+    task.dependsOn(publishToMavenLocalTask)
+  }
+
+  return taskProvider
+}
+
+private fun Project.expoPublishBody(publicationInfo: PublicationInfo, expoModulesExtension: ExpoModuleExtension, pathToRepository: String? = null) {
+  validateProjectConfiguration(expoModulesExtension)
+
+  if (pathToRepository == null) {
+    val mavenLocal = publishingExtension().repositories.mavenLocal()
+    val mavenLocalPath = mavenLocal.url.toPath()
+    val publicationPath = publicationInfo.resolvePath(mavenLocalPath)
+
+    if (!publicationPath.exists()) {
+      return
+    }
+
+    logger.quiet("$publicationInfo was published to $publicationPath")
+  }
+
+  val expoModuleConfig = layout.projectDirectory.file("../expo-module.config.json").asFile
+  val json = Json {
+    ignoreUnknownKeys = true
+    prettyPrint = true
+    prettyPrintIndent = "  "
+  }
+
+  val jsonElement = json.parseToJsonElement(expoModuleConfig.readText()).jsonObject
+  val newJsonElement = modifyModuleConfig(projectName = name, jsonElement, publicationInfo, pathToRepository)
+
+  val newJsonString = json.encodeToString(JsonObject.serializer(), newJsonElement)
+
+  logger.quiet("Updating 'expo-module.config.json' in ${expoModuleConfig.parent}")
+
+  expoModuleConfig.writeText(newJsonString)
+  providers.exec { env ->
+    env.workingDir(layout.projectDirectory.file(".."))
+    // TODO(@lukmccall): support other package managers
+    env.commandLine("yarn", "prettier", "--write", "expo-module.config.json")
+  }.result.get()
+}
+
+private fun Project.validateProjectConfiguration(expoModulesExtension: ExpoModuleExtension) {
+  val shouldUsePublicationScript = expoModulesExtension.autolinking.getShouldUsePublicationScriptPath(this)
+  // If the path to the script is not defined, we assume that we can publish the module.
+  if (shouldUsePublicationScript == null) {
+    return
+  }
+
+  val binding = Binding()
+  binding.setVariable("providers", project.providers)
+  val shell = GroovyShell(javaClass.classLoader, binding)
+
+  val shouldUsePublication = shell.run(shouldUsePublicationScript, emptyArray<String>()) as? Boolean == true
+
+  if (!shouldUsePublication) {
+    throw IllegalStateException("The publication script returned false. Please check the script or your project configuration. You're trying to precompile a non-default configuration.")
+  }
+}
+
+private fun modifyModuleConfig(projectName: String, currentConfig: JsonObject, publicationInfo: PublicationInfo, pathToRepository: String?): JsonObject {
+  val publicationObject = JsonObject(mapOf(
+    "groupId" to publicationInfo.groupId.toJsonElement(),
+    "artifactId" to publicationInfo.artifactId.toJsonElement(),
+    "version" to publicationInfo.version.toJsonElement(),
+    "repository" to (pathToRepository ?: "mavenLocal").toJsonElement(),
+  ))
+
+  val androidObject = currentConfig.getOrDefault("android", JsonObject(emptyMap())).jsonObject.mutate {
+    val subProject = get("projects")
+      ?.jsonArray
+      ?.mapIndexed { index, element -> index to element.jsonObject }
+      ?.find { (_, element) -> element["name"]?.jsonPrimitive?.content == projectName }
+    if (subProject != null) {
+      val (index, project) = subProject
+      val newProject = project.mutate {
+        put("publication", publicationObject)
+      }
+
+      val newProjects = requireNotNull(get("projects"))
+        .jsonArray
+        .toMutableList()
+        .apply {
+          set(index, newProject)
+        }
+        .toJsonArray()
+
+      put("projects", newProjects)
+    } else {
+      put("publication", publicationObject)
+    }
+  }
+
+  return currentConfig.mutate {
+    put("android", androidObject)
+  }
+}
+
+private fun String.toJsonElement(): JsonElement = JsonPrimitive(this)
+private fun Map<String, JsonElement>.toJsonObject(): JsonObject = JsonObject(this)
+private inline fun JsonObject.mutate(block: MutableMap<String, JsonElement>.() -> Unit) =
+  toMutableMap().apply {
+    block()
+  }.toJsonObject()
+
+private fun MutableList<JsonElement>.toJsonArray(): JsonArray = JsonArray(this)
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/bin/main/expo/modules/plugin/gradle/ExpoGradleHelperExtension.kt b/node_modules/expo-modules-core/expo-module-gradle-plugin/bin/main/expo/modules/plugin/gradle/ExpoGradleHelperExtension.kt
new file mode 100644
index 0000000..7c387ab
--- /dev/null
+++ b/node_modules/expo-modules-core/expo-module-gradle-plugin/bin/main/expo/modules/plugin/gradle/ExpoGradleHelperExtension.kt
@@ -0,0 +1,77 @@
+package expo.modules.plugin.gradle
+
+import expo.modules.plugin.Version
+import org.gradle.api.Project
+import java.io.File
+import java.io.FileInputStream
+import java.util.Properties
+
+/**
+ * An extension for the Gradle instance that stores data in a cache,
+ * reducing the number of times it needs to be recomputed.
+ *
+ * For example, to resolve the React Native directory, we need to run an external process.
+ * This can add up - it's better to run it once and cache the result.
+ */
+open class ExpoGradleHelperExtension {
+  /**
+   * Cached React Native directory
+   */
+  private lateinit var reactNativeDir: File
+
+  /**
+   * Cached React Native properties
+   */
+  private lateinit var reactNativeProperties: Properties
+
+  /**
+   * Cached React Native version
+   */
+  private lateinit var reactNativeVersion: Version
+
+  fun getReactNativeDir(project: Project): File = synchronized(this) {
+    if (::reactNativeDir.isInitialized) {
+      return reactNativeDir
+    }
+
+    // When building from source, the ReactAndroid project is available
+    val reactNativeDirFromSource = project
+      .findProject(":packages:react-native:ReactAndroid")
+      ?.projectDir
+      ?.parentFile
+
+    reactNativeDir = reactNativeDirFromSource ?: File(
+      project.providers.exec { env ->
+        env.workingDir(project.rootDir)
+        env.commandLine("node", "--print", "require.resolve('react-native/package.json')")
+      }.standardOutput.asText.get().trim()
+    ).parentFile
+
+    return reactNativeDir
+  }
+
+  fun getReactNativeProperties(project: Project): Properties = synchronized(this) {
+    if (::reactNativeProperties.isInitialized) {
+      return reactNativeProperties
+    }
+
+    val reactPropertiesFile = File(getReactNativeDir(project), "ReactAndroid/gradle.properties")
+    reactNativeProperties = Properties().also { properties ->
+      FileInputStream(reactPropertiesFile).use {
+        properties.load(it)
+      }
+      properties.load(reactPropertiesFile.inputStream())
+    }
+    return reactNativeProperties
+  }
+
+  fun getReactNativeVersion(project: Project): Version = synchronized(this) {
+    if (::reactNativeVersion.isInitialized) {
+      return reactNativeVersion
+    }
+
+    val version = getReactNativeProperties(project).getProperty("VERSION_NAME")
+    reactNativeVersion = Version.fromString(version)
+    return reactNativeVersion
+  }
+}
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/bin/main/expo/modules/plugin/gradle/ExpoModuleExtension.kt b/node_modules/expo-modules-core/expo-module-gradle-plugin/bin/main/expo/modules/plugin/gradle/ExpoModuleExtension.kt
new file mode 100644
index 0000000..896cc50
--- /dev/null
+++ b/node_modules/expo-modules-core/expo-module-gradle-plugin/bin/main/expo/modules/plugin/gradle/ExpoModuleExtension.kt
@@ -0,0 +1,42 @@
+package expo.modules.plugin.gradle
+
+import expo.modules.plugin.AutolinkingIntegration
+import expo.modules.plugin.AutolinkingIntegrationImpl
+import org.gradle.api.Project
+import java.io.File
+import java.util.Properties
+import expo.modules.plugin.Version
+import expo.modules.plugin.safeGet
+import org.gradle.internal.extensions.core.extra
+
+/**
+ * An user-facing interface to interact with the `ExpoGradleHelperExtension`.
+ */
+open class ExpoModuleExtension(val project: Project) {
+  private val gradleHelper by lazy {
+    project.gradle.extensions.getByType(ExpoGradleHelperExtension::class.java)
+  }
+
+  internal val autolinking: AutolinkingIntegration by lazy {
+    AutolinkingIntegrationImpl()
+  }
+
+  val reactNativeDir: File
+    get() = gradleHelper.getReactNativeDir(project)
+
+  val reactNativeProperties: Properties
+    get() = gradleHelper.getReactNativeProperties(project)
+
+  val reactNativeVersion: Version
+    get() = gradleHelper.getReactNativeVersion(project)
+
+  fun safeExtGet(name: String, default: Any): Any {
+    return project.rootProject.extra.safeGet<Any>(name) ?: default
+  }
+
+  fun getExpoDependency(name: String): Any {
+    return autolinking.getExpoDependency(project, name)
+  }
+
+  var canBePublished: Boolean = true
+}
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/classes/kotlin/main/META-INF/expo-module-gradle-plugin.kotlin_module b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/classes/kotlin/main/META-INF/expo-module-gradle-plugin.kotlin_module
new file mode 100644
index 0000000..7c85697
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/classes/kotlin/main/META-INF/expo-module-gradle-plugin.kotlin_module differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/classes/kotlin/main/expo/modules/plugin/AutolinkingIntegration.class b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/classes/kotlin/main/expo/modules/plugin/AutolinkingIntegration.class
new file mode 100644
index 0000000..27ff905
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/classes/kotlin/main/expo/modules/plugin/AutolinkingIntegration.class differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/classes/kotlin/main/expo/modules/plugin/AutolinkingIntegrationImpl.class b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/classes/kotlin/main/expo/modules/plugin/AutolinkingIntegrationImpl.class
new file mode 100644
index 0000000..586bfef
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/classes/kotlin/main/expo/modules/plugin/AutolinkingIntegrationImpl.class differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/classes/kotlin/main/expo/modules/plugin/ExpoModulesGradlePlugin$apply$1$1.class b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/classes/kotlin/main/expo/modules/plugin/ExpoModulesGradlePlugin$apply$1$1.class
new file mode 100644
index 0000000..da20043
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/classes/kotlin/main/expo/modules/plugin/ExpoModulesGradlePlugin$apply$1$1.class differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/classes/kotlin/main/expo/modules/plugin/ExpoModulesGradlePlugin.class b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/classes/kotlin/main/expo/modules/plugin/ExpoModulesGradlePlugin.class
new file mode 100644
index 0000000..adfad1f
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/classes/kotlin/main/expo/modules/plugin/ExpoModulesGradlePlugin.class differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/classes/kotlin/main/expo/modules/plugin/ExpoModulesGradlePluginKt.class b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/classes/kotlin/main/expo/modules/plugin/ExpoModulesGradlePluginKt.class
new file mode 100644
index 0000000..fcf0826
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/classes/kotlin/main/expo/modules/plugin/ExpoModulesGradlePluginKt.class differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/classes/kotlin/main/expo/modules/plugin/ExtraPropertiesExtensionKt.class b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/classes/kotlin/main/expo/modules/plugin/ExtraPropertiesExtensionKt.class
new file mode 100644
index 0000000..17196f3
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/classes/kotlin/main/expo/modules/plugin/ExtraPropertiesExtensionKt.class differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/classes/kotlin/main/expo/modules/plugin/ProjectConfigurationKt.class b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/classes/kotlin/main/expo/modules/plugin/ProjectConfigurationKt.class
new file mode 100644
index 0000000..86e380d
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/classes/kotlin/main/expo/modules/plugin/ProjectConfigurationKt.class differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/classes/kotlin/main/expo/modules/plugin/Version$Companion.class b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/classes/kotlin/main/expo/modules/plugin/Version$Companion.class
new file mode 100644
index 0000000..6577a9c
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/classes/kotlin/main/expo/modules/plugin/Version$Companion.class differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/classes/kotlin/main/expo/modules/plugin/Version.class b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/classes/kotlin/main/expo/modules/plugin/Version.class
new file mode 100644
index 0000000..fb44b24
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/classes/kotlin/main/expo/modules/plugin/Version.class differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/classes/kotlin/main/expo/modules/plugin/WarningsKt.class b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/classes/kotlin/main/expo/modules/plugin/WarningsKt.class
new file mode 100644
index 0000000..47d18d9
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/classes/kotlin/main/expo/modules/plugin/WarningsKt.class differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/classes/kotlin/main/expo/modules/plugin/android/AndroidLibraryExtensionKt.class b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/classes/kotlin/main/expo/modules/plugin/android/AndroidLibraryExtensionKt.class
new file mode 100644
index 0000000..4d731b7
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/classes/kotlin/main/expo/modules/plugin/android/AndroidLibraryExtensionKt.class differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/classes/kotlin/main/expo/modules/plugin/android/MavenPublicationExtensionKt.class b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/classes/kotlin/main/expo/modules/plugin/android/MavenPublicationExtensionKt.class
new file mode 100644
index 0000000..ff68c5a
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/classes/kotlin/main/expo/modules/plugin/android/MavenPublicationExtensionKt.class differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/classes/kotlin/main/expo/modules/plugin/android/PublicationInfo.class b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/classes/kotlin/main/expo/modules/plugin/android/PublicationInfo.class
new file mode 100644
index 0000000..f5c5ef8
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/classes/kotlin/main/expo/modules/plugin/android/PublicationInfo.class differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/classes/kotlin/main/expo/modules/plugin/gradle/ExpoGradleHelperExtension.class b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/classes/kotlin/main/expo/modules/plugin/gradle/ExpoGradleHelperExtension.class
new file mode 100644
index 0000000..c109bfc
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/classes/kotlin/main/expo/modules/plugin/gradle/ExpoGradleHelperExtension.class differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/classes/kotlin/main/expo/modules/plugin/gradle/ExpoModuleExtension.class b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/classes/kotlin/main/expo/modules/plugin/gradle/ExpoModuleExtension.class
new file mode 100644
index 0000000..029e333
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/classes/kotlin/main/expo/modules/plugin/gradle/ExpoModuleExtension.class differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/inputs/source-to-output.tab b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/inputs/source-to-output.tab
new file mode 100644
index 0000000..daebfba
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/inputs/source-to-output.tab differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/inputs/source-to-output.tab.keystream b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/inputs/source-to-output.tab.keystream
new file mode 100644
index 0000000..22518fb
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/inputs/source-to-output.tab.keystream differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/inputs/source-to-output.tab.keystream.len b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/inputs/source-to-output.tab.keystream.len
new file mode 100644
index 0000000..666fb56
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/inputs/source-to-output.tab.keystream.len differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/inputs/source-to-output.tab.len b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/inputs/source-to-output.tab.len
new file mode 100644
index 0000000..a541356
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/inputs/source-to-output.tab.len differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/inputs/source-to-output.tab.values.at b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/inputs/source-to-output.tab.values.at
new file mode 100644
index 0000000..7e4a04e
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/inputs/source-to-output.tab.values.at differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/inputs/source-to-output.tab_i b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/inputs/source-to-output.tab_i
new file mode 100644
index 0000000..d6a146a
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/inputs/source-to-output.tab_i differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/inputs/source-to-output.tab_i.len b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/inputs/source-to-output.tab_i.len
new file mode 100644
index 0000000..131e265
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/inputs/source-to-output.tab_i.len differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/class-attributes.tab b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/class-attributes.tab
new file mode 100644
index 0000000..fb174fd
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/class-attributes.tab differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/class-attributes.tab.keystream b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/class-attributes.tab.keystream
new file mode 100644
index 0000000..b49e701
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/class-attributes.tab.keystream differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/class-attributes.tab.keystream.len b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/class-attributes.tab.keystream.len
new file mode 100644
index 0000000..2553f37
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/class-attributes.tab.keystream.len differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/class-attributes.tab.len b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/class-attributes.tab.len
new file mode 100644
index 0000000..fa606b6
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/class-attributes.tab.len differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/class-attributes.tab.values.at b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/class-attributes.tab.values.at
new file mode 100644
index 0000000..de0b0a4
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/class-attributes.tab.values.at differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/class-attributes.tab_i b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/class-attributes.tab_i
new file mode 100644
index 0000000..32ff494
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/class-attributes.tab_i differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/class-attributes.tab_i.len b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/class-attributes.tab_i.len
new file mode 100644
index 0000000..131e265
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/class-attributes.tab_i.len differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/class-fq-name-to-source.tab b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/class-fq-name-to-source.tab
new file mode 100644
index 0000000..fb06e23
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/class-fq-name-to-source.tab differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/class-fq-name-to-source.tab.keystream b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/class-fq-name-to-source.tab.keystream
new file mode 100644
index 0000000..b49e701
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/class-fq-name-to-source.tab.keystream differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/class-fq-name-to-source.tab.keystream.len b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/class-fq-name-to-source.tab.keystream.len
new file mode 100644
index 0000000..2553f37
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/class-fq-name-to-source.tab.keystream.len differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/class-fq-name-to-source.tab.len b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/class-fq-name-to-source.tab.len
new file mode 100644
index 0000000..fa606b6
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/class-fq-name-to-source.tab.len differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/class-fq-name-to-source.tab.values.at b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/class-fq-name-to-source.tab.values.at
new file mode 100644
index 0000000..dab39d1
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/class-fq-name-to-source.tab.values.at differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/class-fq-name-to-source.tab_i b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/class-fq-name-to-source.tab_i
new file mode 100644
index 0000000..32ff494
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/class-fq-name-to-source.tab_i differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/class-fq-name-to-source.tab_i.len b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/class-fq-name-to-source.tab_i.len
new file mode 100644
index 0000000..131e265
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/class-fq-name-to-source.tab_i.len differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/inline-functions.tab b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/inline-functions.tab
new file mode 100644
index 0000000..bdf584a
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/inline-functions.tab differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/inline-functions.tab.keystream b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/inline-functions.tab.keystream
new file mode 100644
index 0000000..75ada3e
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/inline-functions.tab.keystream differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/inline-functions.tab.keystream.len b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/inline-functions.tab.keystream.len
new file mode 100644
index 0000000..2409504
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/inline-functions.tab.keystream.len differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/inline-functions.tab.len b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/inline-functions.tab.len
new file mode 100644
index 0000000..2a17e6e
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/inline-functions.tab.len differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/inline-functions.tab.values.at b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/inline-functions.tab.values.at
new file mode 100644
index 0000000..27d01a4
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/inline-functions.tab.values.at differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/inline-functions.tab_i b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/inline-functions.tab_i
new file mode 100644
index 0000000..0a6a825
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/inline-functions.tab_i differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/inline-functions.tab_i.len b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/inline-functions.tab_i.len
new file mode 100644
index 0000000..131e265
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/inline-functions.tab_i.len differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/internal-name-to-source.tab b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/internal-name-to-source.tab
new file mode 100644
index 0000000..a6ca7fa
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/internal-name-to-source.tab differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/internal-name-to-source.tab.keystream b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/internal-name-to-source.tab.keystream
new file mode 100644
index 0000000..732360d
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/internal-name-to-source.tab.keystream differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/internal-name-to-source.tab.keystream.len b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/internal-name-to-source.tab.keystream.len
new file mode 100644
index 0000000..d3545ea
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/internal-name-to-source.tab.keystream.len differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/internal-name-to-source.tab.len b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/internal-name-to-source.tab.len
new file mode 100644
index 0000000..cf8a30a
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/internal-name-to-source.tab.len differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/internal-name-to-source.tab.values.at b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/internal-name-to-source.tab.values.at
new file mode 100644
index 0000000..11f30a0
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/internal-name-to-source.tab.values.at differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/internal-name-to-source.tab_i b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/internal-name-to-source.tab_i
new file mode 100644
index 0000000..ebbe9dc
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/internal-name-to-source.tab_i differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/internal-name-to-source.tab_i.len b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/internal-name-to-source.tab_i.len
new file mode 100644
index 0000000..131e265
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/internal-name-to-source.tab_i.len differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/package-parts.tab b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/package-parts.tab
new file mode 100644
index 0000000..22f188f
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/package-parts.tab differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/package-parts.tab.keystream b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/package-parts.tab.keystream
new file mode 100644
index 0000000..4ee0242
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/package-parts.tab.keystream differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/package-parts.tab.keystream.len b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/package-parts.tab.keystream.len
new file mode 100644
index 0000000..856c838
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/package-parts.tab.keystream.len differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/package-parts.tab.len b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/package-parts.tab.len
new file mode 100644
index 0000000..9e27f73
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/package-parts.tab.len differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/package-parts.tab.values.at b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/package-parts.tab.values.at
new file mode 100644
index 0000000..d06c58f
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/package-parts.tab.values.at differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/package-parts.tab_i b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/package-parts.tab_i
new file mode 100644
index 0000000..0633ce3
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/package-parts.tab_i differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/package-parts.tab_i.len b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/package-parts.tab_i.len
new file mode 100644
index 0000000..131e265
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/package-parts.tab_i.len differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/proto.tab b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/proto.tab
new file mode 100644
index 0000000..0fd0760
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/proto.tab differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/proto.tab.keystream b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/proto.tab.keystream
new file mode 100644
index 0000000..a01dd33
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/proto.tab.keystream differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/proto.tab.keystream.len b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/proto.tab.keystream.len
new file mode 100644
index 0000000..788f6e7
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/proto.tab.keystream.len differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/proto.tab.len b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/proto.tab.len
new file mode 100644
index 0000000..cf8a30a
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/proto.tab.len differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/proto.tab.values.at b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/proto.tab.values.at
new file mode 100644
index 0000000..d7a6ebe
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/proto.tab.values.at differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/proto.tab_i b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/proto.tab_i
new file mode 100644
index 0000000..1e373b7
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/proto.tab_i differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/proto.tab_i.len b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/proto.tab_i.len
new file mode 100644
index 0000000..131e265
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/proto.tab_i.len differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/source-to-classes.tab b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/source-to-classes.tab
new file mode 100644
index 0000000..a13605a
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/source-to-classes.tab differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/source-to-classes.tab.keystream b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/source-to-classes.tab.keystream
new file mode 100644
index 0000000..22518fb
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/source-to-classes.tab.keystream differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/source-to-classes.tab.keystream.len b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/source-to-classes.tab.keystream.len
new file mode 100644
index 0000000..666fb56
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/source-to-classes.tab.keystream.len differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/source-to-classes.tab.len b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/source-to-classes.tab.len
new file mode 100644
index 0000000..a541356
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/source-to-classes.tab.len differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/source-to-classes.tab.values.at b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/source-to-classes.tab.values.at
new file mode 100644
index 0000000..2c7e06d
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/source-to-classes.tab.values.at differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/source-to-classes.tab_i b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/source-to-classes.tab_i
new file mode 100644
index 0000000..d6a146a
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/source-to-classes.tab_i differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/source-to-classes.tab_i.len b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/source-to-classes.tab_i.len
new file mode 100644
index 0000000..131e265
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/source-to-classes.tab_i.len differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/subtypes.tab b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/subtypes.tab
new file mode 100644
index 0000000..5dbc487
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/subtypes.tab differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/subtypes.tab.keystream b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/subtypes.tab.keystream
new file mode 100644
index 0000000..49d70c3
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/subtypes.tab.keystream differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/subtypes.tab.keystream.len b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/subtypes.tab.keystream.len
new file mode 100644
index 0000000..a8206a1
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/subtypes.tab.keystream.len differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/subtypes.tab.len b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/subtypes.tab.len
new file mode 100644
index 0000000..01bdaa1
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/subtypes.tab.len differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/subtypes.tab.values.at b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/subtypes.tab.values.at
new file mode 100644
index 0000000..00bcfdc
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/subtypes.tab.values.at differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/subtypes.tab_i b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/subtypes.tab_i
new file mode 100644
index 0000000..56c7ed1
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/subtypes.tab_i differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/subtypes.tab_i.len b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/subtypes.tab_i.len
new file mode 100644
index 0000000..131e265
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/subtypes.tab_i.len differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/supertypes.tab b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/supertypes.tab
new file mode 100644
index 0000000..a25e8e9
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/supertypes.tab differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/supertypes.tab.keystream b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/supertypes.tab.keystream
new file mode 100644
index 0000000..3214a6f
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/supertypes.tab.keystream differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/supertypes.tab.keystream.len b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/supertypes.tab.keystream.len
new file mode 100644
index 0000000..8790089
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/supertypes.tab.keystream.len differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/supertypes.tab.len b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/supertypes.tab.len
new file mode 100644
index 0000000..01bdaa1
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/supertypes.tab.len differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/supertypes.tab.values.at b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/supertypes.tab.values.at
new file mode 100644
index 0000000..e15fbd6
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/supertypes.tab.values.at differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/supertypes.tab_i b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/supertypes.tab_i
new file mode 100644
index 0000000..84e6a94
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/supertypes.tab_i differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/supertypes.tab_i.len b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/supertypes.tab_i.len
new file mode 100644
index 0000000..131e265
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/jvm/kotlin/supertypes.tab_i.len differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/lookups/counters.tab b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/lookups/counters.tab
new file mode 100644
index 0000000..920202f
--- /dev/null
+++ b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/lookups/counters.tab
@@ -0,0 +1,2 @@
+11
+0
\ No newline at end of file
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/lookups/file-to-id.tab b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/lookups/file-to-id.tab
new file mode 100644
index 0000000..1b64afe
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/lookups/file-to-id.tab differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/lookups/file-to-id.tab.keystream b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/lookups/file-to-id.tab.keystream
new file mode 100644
index 0000000..22518fb
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/lookups/file-to-id.tab.keystream differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/lookups/file-to-id.tab.keystream.len b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/lookups/file-to-id.tab.keystream.len
new file mode 100644
index 0000000..666fb56
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/lookups/file-to-id.tab.keystream.len differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/lookups/file-to-id.tab.len b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/lookups/file-to-id.tab.len
new file mode 100644
index 0000000..a541356
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/lookups/file-to-id.tab.len differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/lookups/file-to-id.tab.values.at b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/lookups/file-to-id.tab.values.at
new file mode 100644
index 0000000..afce831
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/lookups/file-to-id.tab.values.at differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/lookups/file-to-id.tab_i b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/lookups/file-to-id.tab_i
new file mode 100644
index 0000000..d6a146a
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/lookups/file-to-id.tab_i differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/lookups/file-to-id.tab_i.len b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/lookups/file-to-id.tab_i.len
new file mode 100644
index 0000000..131e265
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/lookups/file-to-id.tab_i.len differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/lookups/id-to-file.tab b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/lookups/id-to-file.tab
new file mode 100644
index 0000000..b4a40de
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/lookups/id-to-file.tab differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/lookups/id-to-file.tab.keystream b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/lookups/id-to-file.tab.keystream
new file mode 100644
index 0000000..b73411e
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/lookups/id-to-file.tab.keystream differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/lookups/id-to-file.tab.keystream.len b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/lookups/id-to-file.tab.keystream.len
new file mode 100644
index 0000000..296694d
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/lookups/id-to-file.tab.keystream.len differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/lookups/id-to-file.tab.len b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/lookups/id-to-file.tab.len
new file mode 100644
index 0000000..a541356
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/lookups/id-to-file.tab.len differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/lookups/id-to-file.tab.values.at b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/lookups/id-to-file.tab.values.at
new file mode 100644
index 0000000..6346e37
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/lookups/id-to-file.tab.values.at differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/lookups/id-to-file.tab_i b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/lookups/id-to-file.tab_i
new file mode 100644
index 0000000..fd27401
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/lookups/id-to-file.tab_i differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/lookups/id-to-file.tab_i.len b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/lookups/id-to-file.tab_i.len
new file mode 100644
index 0000000..131e265
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/lookups/id-to-file.tab_i.len differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/lookups/lookups.tab b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/lookups/lookups.tab
new file mode 100644
index 0000000..b7eafe5
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/lookups/lookups.tab differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/lookups/lookups.tab.keystream b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/lookups/lookups.tab.keystream
new file mode 100644
index 0000000..bcc8151
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/lookups/lookups.tab.keystream differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/lookups/lookups.tab.keystream.len b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/lookups/lookups.tab.keystream.len
new file mode 100644
index 0000000..b509e6e
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/lookups/lookups.tab.keystream.len differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/lookups/lookups.tab.len b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/lookups/lookups.tab.len
new file mode 100644
index 0000000..1d114f0
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/lookups/lookups.tab.len differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/lookups/lookups.tab.values.at b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/lookups/lookups.tab.values.at
new file mode 100644
index 0000000..4a294dd
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/lookups/lookups.tab.values.at differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/lookups/lookups.tab_i b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/lookups/lookups.tab_i
new file mode 100644
index 0000000..4438f61
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/lookups/lookups.tab_i differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/lookups/lookups.tab_i.len b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/lookups/lookups.tab_i.len
new file mode 100644
index 0000000..131e265
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/caches-jvm/lookups/lookups.tab_i.len differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/last-build.bin b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/last-build.bin
new file mode 100644
index 0000000..12faa0b
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/cacheable/last-build.bin differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/classpath-snapshot/shrunk-classpath-snapshot.bin b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/classpath-snapshot/shrunk-classpath-snapshot.bin
new file mode 100644
index 0000000..0d0fe34
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/kotlin/compileKotlin/classpath-snapshot/shrunk-classpath-snapshot.bin differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/libs/expo-module-gradle-plugin.jar b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/libs/expo-module-gradle-plugin.jar
new file mode 100644
index 0000000..770d01d
Binary files /dev/null and b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/libs/expo-module-gradle-plugin.jar differ
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/pluginDescriptors/expo-module-gradle-plugin.properties b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/pluginDescriptors/expo-module-gradle-plugin.properties
new file mode 100644
index 0000000..08df140
--- /dev/null
+++ b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/pluginDescriptors/expo-module-gradle-plugin.properties
@@ -0,0 +1 @@
+implementation-class=expo.modules.plugin.ExpoModulesGradlePlugin
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/resources/main/META-INF/gradle-plugins/expo-module-gradle-plugin.properties b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/resources/main/META-INF/gradle-plugins/expo-module-gradle-plugin.properties
new file mode 100644
index 0000000..08df140
--- /dev/null
+++ b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/resources/main/META-INF/gradle-plugins/expo-module-gradle-plugin.properties
@@ -0,0 +1 @@
+implementation-class=expo.modules.plugin.ExpoModulesGradlePlugin
diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build/tmp/jar/MANIFEST.MF b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/tmp/jar/MANIFEST.MF
new file mode 100644
index 0000000..59499bc
--- /dev/null
+++ b/node_modules/expo-modules-core/expo-module-gradle-plugin/build/tmp/jar/MANIFEST.MF
@@ -0,0 +1,2 @@
+Manifest-Version: 1.0
+
