diff --git a/node_modules/react-native-gesture-handler/android/src/main/jni/CMakeLists.txt b/node_modules/react-native-gesture-handler/android/src/main/jni/CMakeLists.txt
index 026e7ca..810ebfa 100644
--- a/node_modules/react-native-gesture-handler/android/src/main/jni/CMakeLists.txt
+++ b/node_modules/react-native-gesture-handler/android/src/main/jni/CMakeLists.txt
@@ -38,4 +38,5 @@ target_link_libraries(
   ReactAndroid::reactnative
   ReactAndroid::jsi
   fbjni::fbjni
+  c++_shared
 )
